odoo/models.py

```diff
@@ -3511,7 +3511,7 @@ def qualify(field):
             if forbidden:
                 raise self.env['ir.rule']._make_access_error('read', forbidden)
 
-    def get_metadata(self):
+    def get_metadata(self, with_xmlids=False):
```

Comment:
Docstring needs to document the new parameter, and new key.

Reply:
removed with new implementation